spec/features/research_article_reviewer_report_spec.rb

```diff
@@ -261,8 +261,7 @@ def create_reviewer_report_task
       wait_for_editors
       set_rich_text editor: 'reviewer_report--competing_interests--detail', text: 'save this'
       find('.required-standalone textarea').set 'Edit notes'
-      wait_for_ajax
-      expect(Answer.first.value).to eq '<p>I have no competing interests</p>'
+      expect { Answer.first.value }.to become('<p>I have no competing interests</p>')
```

Comment:
`Answer.first` wont cache right?

Reply:
you know I was wondering that

Reply:
no
```
[5] pry(main)> User.first.object_id
=> 46963211519280
[6] pry(main)> User.first.object_id
=> 46963212048300
```

Reply:
does it cache the sql query though?

Reply:
```
require "rails_helper"

RSpec.describe 'misc' do
  let!(:journal) { FactoryGirl.create(:journal) }

  it 'should not cache db queries' do
    expect do
      Journal.first.name
      Journal.first.name
      Journal.first.name
    end.to make_database_queries(count: 3)
  end
end
```
```
Finished in 0.04366 seconds (files took 9.92 seconds to load)
1 example, 0 failures
```

No.

Reply:
And double checking the log:
```
  Journal Load (0.8ms)  SELECT  "journals".* FROM "journals"  ORDER BY "journals"."id" ASC LIMIT 1
  Journal Load (0.5ms)  SELECT  "journals".* FROM "journals"  ORDER BY "journals"."id" ASC LIMIT 1
  Journal Load (0.4ms)  SELECT  "journals".* FROM "journals"  ORDER BY "journals"."id" ASC LIMIT 1
```

Reply:
Nice, thanks for checking. To this day, I'm still woefully uninformed about how AR SQL queries are cached. Thankfully it appears to work how I'd expect in rspec--that is, it doesn't cache.

Reply:
I believe this is because controller caching is disabled in test env

Reply:
right, and this is also outside of a controller.